ocamlformat.opam

```diff
@@ -28,7 +28,7 @@ depends: [
   "cmdliner"
   "dune" {build & >= "1.1.1"}
   "fpath"
-  "ocaml-migrate-parsetree" {>= "1.0.10"}
+  "ocaml-migrate-parsetree" {>= "1.3.0"}
```

Comment:
will probably be 1.3.1 (see https://github.com/ocaml-ppx/ocaml-migrate-parsetree/issues/73)